src/components/EventModal/EventOwner.tsx

```diff
@@ -131,6 +180,63 @@ export default function EventOwner(props: EventOwnerProps) {
           Add
         </Button>
       </Grid>
+      {scannerOpen ? (
```

Comment:
Also, only show the ability to scan if the event is active and the event end date hasn't passed